### PR TITLE
adds a parameter which modifies the kinematics to the test case. 

### LINKED ISSUE #1137: Unable to parameterize joint locations
Is parameterizing joint locations something that is supported?  For instance, If I want to parse the following URDF:
```
<?xml version="1.0"?>

<robot xmlns="http://drake.mit.edu"
 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 xsi:schemaLocation="http://drake.mit.edu ../../../doc/drakeURDF.xsd" name="Acrobot">

  <parameter name="b1" value=".1" lb="0" />
  <parameter name="b2" value=".1" lb="0" />
  <parameter name="Ic1" value=".083" lb="0" />
  <parameter name="Ic2" value=".33" lb="0" />
  <parameter name="length1" value="1.0" lb="0" />
  <parameter name="length2" value="2.0" lb="0" />

  <link name="base_link">
    <visual>
      <geometry>
         <box size=".2 .2 .2" />
      </geometry>
      <material name="green">
	<color rgba="0 1 0 1" />
      </material>
    </visual>
  </link>

  <link name="upper_link">
    <inertial>
      <origin xyz="0 0 -$length1 / 2" rpy="0 0 0" />
      <mass value="1" />
      <inertia ixx="1" ixy="0" ixz="0" iyy="$Ic1" iyz="0" izz="1" />
    </inertial>
    <visual>
      <origin xyz="0 0 -.5" rpy="0 0 0" />
      <geometry>
         <cylinder length="1.1" radius=".05" />
      </geometry>
      <material name="red">
	<color rgba="1 0 0 1" />
      </material>
    </visual>
  </link>

  <link name="lower_link">
    <inertial>
      <origin xyz="0 0 -$length2 / 2" rpy="0 0 0" />
      <mass value="1" />
      <inertia ixx="1" ixy="0" ixz="0" iyy="$Ic2" iyz="0" izz="1" />
    </inertial>
    <visual>
      <origin xyz="0 0 -1" rpy="0 0 0" />
      <geometry>
         <cylinder length="2.1" radius=".05" />
      </geometry>
      <material name="blue">
	<color rgba="0 0 1 1" />
      </material>
    </visual>
  </link>
  
  <joint name="shoulder" type="continuous">
    <parent link="base_link" />
    <child link="upper_link" />
    <origin xyz="0 .15 0" />
    <axis xyz="0 1 0" />
    <dynamics damping="$b1" />
  </joint>

  <joint name="elbow" type="continuous">
    <parent link="upper_link" />
    <child link="lower_link" />
    <origin xyz="0 .1 -$length1" />
    <axis xyz="0 1 0" />
    <dynamics damping="$b2" />
  </joint>

  <transmission type="SimpleTransmission" name="elbow_trans">
    <actuator name="elbow_torque" />
    <joint name="elbow" />
    <mechanicalReduction>1</mechanicalReduction>
  </transmission>
</robot>
```

Here I have parameterized the origin of the elbow joint.  In essence, I want my robot to be a continuous kinematic chain, and if one of the robot link lengths is a parameter, then the joint location will also have to be a parameter.

When I try to parse this in, however, I get the following error:
```
Undefined function 'any' for input arguments of type 'msspoly'.

Error in RigidBodyManipulator/addJoint (line 451)
      if any(xyz)

Error in RigidBodyManipulator/addRobotFromURDF>parseJoint (line 369)
model =
addJoint(model,name,type,parent,child,xyz,rpy,axis,damping,coulomb_friction,static_friction,coulomb_window,limits);

Error in RigidBodyManipulator/addRobotFromURDF>parseRobot (line 104)
  model = parseJoint(model,robotnum,joints.item(i),options);

Error in RigidBodyManipulator/addRobotFromURDF (line 54)
model = parseRobot(model,robot,xyz,rpy,options);

Error in RigidBodyManipulator (line 72)
          obj = addRobotFromURDF(obj,filename,zeros(3,1),zeros(3,1),options);
```

### PR DESCRIPTION
**Pull request by RussTedrake**
fixes a bug in passing msspoly's through the old wrl parser.
adds an extra test to acrobot parameter test.
resolves #1137

### CODE REVIEW

**Comment by RussTedrake** [comment]
also resolves #1136.

File: systems/plants/viewer/drake_urdf_renderer.cpp [hunk 1]
```diff
@@ -345,7 +345,7 @@ static void handle_lcm_viewer_load_robot(const lcm_recv_buf_t *rbuf, const char
   drake_lcmt_viewer_command_publish(self->lcm, "DRAKE_VIEWER_STATUS", &status_message);
   cout << "successfully loaded model" << endl;
 
-  bot_viewer_request_redraw(self->viewer);
+  // bot_viewer_request_redraw(self->viewer);
```
Comment (psiorx) [inline systems/plants/viewer/drake_urdf_renderer.cpp]: Was this not needed?

File: examples/Acrobot/test/AcrobotWParams.urdf [hunk 2]
```diff
@@ -6,6 +6,7 @@
 
   <parameter name="b1" value=".1" lb="0" />
   <parameter name="b2" value=".1" lb="0" />
+  <parameter name="l1" value="1" lb="0" />
   <parameter name="lc1" value=".5" lb="0" />
```
Comment (psiorx) [inline examples/Acrobot/test/AcrobotWParams.urdf]: Would it make sense to have the center of mass distance scale with the length of the arm?  We could zap lc1 and just have the CoM distance be computed as l1/2.

**Comment by psiorx** [comment]
+1

It's fun seeing the acrobot arm length change at runtime.

File: systems/plants/viewer/drake_urdf_renderer.cpp (hunk 1 above)
Comment (RussTedrake) [inline systems/plants/viewer/drake_urdf_renderer.cpp]: We had a draw command on load, but it does not have a known posture, so always drew at q=0 .  That was bad for the new test case.  Removing it did not seem to have any negative impact when i ran a bunch of the ither examples.  I think it's better out

**Pull request merged**